# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/devfsd/head-tail.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur devfsd/GNUmakefile devfsd-p/GNUmakefile
--- devfsd/GNUmakefile	2002-02-19 07:06:43.000000000 +0100
+++ devfsd-p/GNUmakefile	2004-11-04 20:02:02.042183088 +0100
@@ -10,7 +10,7 @@
 
 nsl_libs := $(wildcard /lib/libnsl.so.*)
 ifneq ($(nsl_libs),)
-LIBNSL      = $(shell for file in /lib/libnsl.so.[0-9]* ; do echo $$file ; done | sort -t . -n -k 2 | head -1)
+LIBNSL      = $(shell for file in /lib/libnsl.so.[0-9]* ; do echo $$file ; done | sort -t . -n -k 2 | head -n1)
 DEFINES     += -DLIBNSL="\"$(LIBNSL)\""
 endif
 
@@ -45,7 +45,7 @@
 
 rpm.spec:	version.h
 	echo "%define version `grep '#define DEVFSD_VERSION' version.h | cut -d'"' -f 2`" > rpm.new
-	tail +2 rpm.spec >> rpm.new
+	tail -n+2 rpm.spec >> rpm.new
 	mv rpm.new rpm.spec
 
 
Nur in devfsd-p/: GNUmakefile~.
